doc/tutorial/tutorial.mdk

```diff
@@ -1308,6 +1357,14 @@ By adding a type annotation we can give a _different_ type to `map`:
 This type is neither a subtype nor a supertype of the type F\*
 inferred on its own.
 
+~KK
+At this point I could not really figure out how the type inferred by F\* and the
+explicitly written type differ functionally. I suspect this is due to my limited
+knowledge of functional programming and has something to do with currying.
+Anyway, a small remark about this might make sense here.
+Also: why can or does F\* not infer the different type? 
+~
+

```

Comment:
I think what I meant was the difference between the types
('a -> ML 'b) -> Tot (list 'a -> ML (list 'b))
and
('a -> Tot 'b) -> list 'a -> Tot (list 'b)
I agree that post- and pre-condition are stronger, but the former type seems to imply that map returns a function that maps a list to another list.